File: Distance Between Bus Stops/Leetcode_1184.cpp
class Solution {
public:
    int distanceBetweenBusStops(vector<int>& distance, int start, int destination) {
        
        if (start > destination) {
            int dummy = start;
            start = destination;
            destination = dummy;
        }
        
        int ck = 0;
        for (int i=start; i < destination; ++i) ck += distance[i];
        int cck = accumulate(distance.begin(), distance.end(), 0) - ck;
        
        return min(ck, cck);
    }
};

File: Distance Between Bus Stops/Leetcode_1184.py
class Solution:
    def distanceBetweenBusStops(self, dist: List[int], start: int, destination: int) -> int:
        n = len(dist)
        if start > destination: start, destination = destination, start
        ck = sum(dist[start:destination])
        cck = sum(dist) - ck
        return min(ck, cck)
